devices/ajax_online.js

```diff
@@ -39,8 +39,9 @@ module.exports = [
         meta: {applyRedFix: true, enhancedHue: false},
     },
     {
+        fingerprint: [{modelID: 'CCT Light', manufacturerName: 'ZB/Ajax Online', manufacturerID: 4137}],
         zigbeeModel: ['ZB-CCT_Filament'],
-        model: 'ZB-CCT_Filament',
+        model: 'CCT Light',
```

Comment:
This is a different device, please create a new definition for it (you can copy paste the existing one and change the `description`)

Reply:
It's the same device, I have added the fingerprint from the converter when I initially created it to show where I think I went wrong.

I believe my error was entering `ZB-CCT_Filament` as the model as listed on the manufacturer website instead of `CCT Light`, which is what is listed in the fingerprint. I didn't realise that model had to match what was in the fingerprint.